collect_app/build.gradle

```diff
@@ -70,6 +70,10 @@ def mapboxToken = secrets.getProperty('MAPBOX_ACCESS_TOKEN', '')
 android {
     compileSdkVersion(28)
 
+    viewBinding {
```

Comment:
It looks like Google have switched up how this should be enabled annoyingly (found [here](https://developer.android.com/topic/libraries/view-binding#setup)). This will throw up a deprecation warning on build. Instead, we need:

```
 android {
    buildFeatures {
        viewBinding true
    }
}
```

Reply:
I did the same initially, but the Gradle sync fails and it says `No signature of method: build_.android() is applicable for argument types`

Reply:
Ah it looks like this PR has the older Gradle plugin. If you update your branch (`git rebase master`) so it has the latest changes from `master` you should be able to use the new definition.

Reply:
Oh sure I will do that @seadowg